apps/tangle-dapp/components/ValidatorSelectionTable/ValidatorSelectionTable.tsx

```diff
@@ -123,84 +115,49 @@ const ValidatorSelectionTable: FC<ValidatorSelectionTableProps> = ({
                   textToCopy={address}
                   isButton={false}
                   className="cursor-pointer"
+                  iconClassName="!fill-mono-160 dark:!fill-mono-80"
                 />
               </div>
             </div>
           );
         },
-        // Sort the selected validators first
-        sortingFn: (rowA, rowB) => {
-          const rowASelected = rowA.getIsSelected();
-          const rowBSelected = rowB.getIsSelected();
-
-          if (rowASelected && !rowBSelected) {
-            return -1;
-          }
-
-          if (!rowASelected && rowBSelected) {
-            return 1;
-          }
-
-          return 0;
-        },
+        sortingFn: sortAddressOrIdentityForNomineeOrValidator,
       }),
       columnHelper.accessor('totalStakeAmount', {
-        header: ({ header }) => (
-          <div
-            className="flex items-center justify-center cursor-pointer"
-            onClick={header.column.getToggleSortingHandler()}
-          >
-            <HeaderCell title="Total Staked" className="flex-none block" />
-
-            <SortArrow column={header.column} />
-          </div>
+        header: () => (
+          <HeaderCell title="Total Staked" className="justify-start" />
```

Comment:
It appears that you have not passed the 'toggleSortingHandler' into this component to enable sorting toggling.

Reply:
The `Total Staked` column is sortable as shown the video